stories/Containers/HorizontalScrollList.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { HorizontalScrollList, ScrollListProps } from '../../src';
+import {createStoryTitle} from "../Constants";
+import { Meta, Story } from '@storybook/react';
+
+const exampleHoveredStyle = () => `
+    color: red;
+`;
+
+const exampleSelectedStyle = () => `
```

Comment:
This worked? Surprised you didn't need to do, css''
I'll prove if you tested though